changelog.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features
 
+- [#2877](https://github.com/ignite/cli/pull/2877) Plugin system
```

Comment:
This is gonna have to move now :)

Reply:
true [1fbb60d](https://github.com/ignite/cli/pull/2877/commits/1fbb60d575c596c54a8bc0a3e8fd1d339036d61e) :)